pkg/oc/admin/router/router.go

```diff
@@ -279,7 +279,7 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out, errout io.
 		},
 	}
 
-	cmd.Flags().StringVar(&cfg.Type, "type", "haproxy-router", "The type of router to use - if you specify --images this flag may be ignored.")
+	cmd.Flags().StringVar(&cfg.Type, "type", "haproxy-router", "The type of router to use, default is haproxy-router - if you specify --images this flag may be ignored.")
```

Comment:
You don't need to describe defaults here, because the default is already printed when you run help

Reply:
OK